scripts/Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG VERSION
-FROM ghcr.io/osgeo/proj:${VERSION:-9.5.0}
+FROM --platform=linux/amd64 ghcr.io/osgeo/proj:${VERSION:-9.5.0}
```

Comment:
Reviewing the logs https://github.com/OSGeo/spatialreference.org/actions/runs/15831285739/job/44766671482#step:6:9 I found this warning
`WARN: FromPlatformFlagConstDisallowed: FROM --platform flag should not use constant value "linux/amd64" (line 2)`

There is some documentation here https://docs.docker.com/reference/build-checks/from-platform-flag-const-disallowed/

Why are you adding it here?

Reply:
This Dockerfile will not work on Apple Silicon unless the platform is specified

Reply:
Sure, done!